core/pages/PageNotFound.js

```diff
@@ -1,32 +0,0 @@
-import { prepareQuery } from '@vue-storefront/core/modules/catalog/queries/common'
```

Comment:
we shouldn't remove this file now, leave it for backward compatibility like the rest of core pages, but just remove dependency from PageNotFound.vue